docs/Developers/Newcomers_Guide.rst

```diff
@@ -298,7 +298,64 @@ Ask a developer to check if it should be reclassified as `low` difficulty.
       please check the following link:
       `PEP8 Style Guide for Python code <https://www.python.org/dev/peps/pep-0008/>`_
 
-Step 3. Creating a Fork and Testing Your Changes
+Step 3. Verifying the Issue Exists
+----------------------------------
+
+This is an important step while taking up an issue which is often overlooked.
+You must ensure that the issue you would like to work on exists beecause it is
+possible that it has already been fixed as a result of a PR for a different
+issue.
+
+.. important::
+
+    * If the issue is a bug in the source code then that shall be confirmed by
+      executing the relevant code or files. You should also figure out how will
+      you rectify this bug.
+
+    * If the issue is about some discrepancy in the documentation, then locate
+      the region in the documentation where it exists. Find out how it can be
+      removed.
+
+    * If the issue requires some additional functionality to be added, then
+      check whether it is possible and does not conflict with the existing
+      ones.
+
+    * If the issue requires the code to have some comments, then provide them
+      in plain and easy-to-understand language.
+
+    * If the issue is some grammatical or spelling error either in comments or
+      documentation, then figure out if the problem still exists and how it can
+      be corrected.
+
+You must verify that the issue actually exists and work on it only if you know
+how to fix it. While verifying the issue, you must start thinking in the
+direction of resolving it as well. If you are able to find a fix for the
+issue, then you can definitely claim it!
+
+The following should also be done to confirm the validity of the issue:
+
+- Check for 'closed' tag on the issue. If you can see the red-coloured 'closed'
+  tag, then it means that the issue is closed. An issue is closed when a pull
+  request resolving that issue is merged into the repository's default branch.
+- Check if an open pull request exists or not. If it exists, then you may look
+  for some other issue because an active PR means that someone else is working
+  on that issue and that person's work is yet to be reviewed. You may also note
+  that even if there is an open pull request, sometimes people can still claim
+  the issue.
+  e.g. the open pull request has been stale for months. Then it is very likely
+  the previous developer either forgot about it or gave up. In this case, you
+  need to claim that you would like to do the issue and also ask for help on
+  our gitter channel.
+- You can also type the issue number in the search bar appearing on the
+  issues page. If the issue exists then it will appear in the search results.
+  Also ensure that it is not assigned to anyone by checking the 'Assignee'
+  column.
+- Newcomers cannot work on issues having label ``initiatives/gci`` as they are
```

Comment:
Make sure the issue you are choosing isn't having label `initiatives/gci` as they are reserved for Google Code-in students contributing to coala. Try finding another [newcomer issue](https://coala.io/new).
I think this will look better ;-)
Verify the syntax of link as this is for `.md` file

Reply:
There is already a link to 'newcomer issues' in step 2 of the Newcomer's guide. And this is step 3. Should I still add the link??

Reply:
I have made the changes. You may review them.

Reply:
'gci' is unrelated to verifying the issue exists.

Reply:
Ya It's unrelated, but I think it should be mentioned somewhere because this guide refers to newcomers and they should be informed that they cannot work on issues that have label `initiatives/gci` on it because sometimes newcomers choose a issue that has a label `initaitives/gci` on it and when that ask for the issue to be assigned to them then they are informed that they can't work on this issue because it is reserved for `gci` students. 
Also we are asking newcomer to check validity of issue by asking him to check `closed` tag etc. So IMO we should mention this in last section as we want a newcomer to choose a valid issue for him.

Reply:
so shall I include this point or not?? please discuss it amongst yourselves and let me know.